build/main.go

```diff
@@ -44,6 +44,8 @@ func main() {
 	flag.Usage = usage
 	flag.Parse()
 
+	goyek.UseExecutor(middleware.ReportFlow)
```

Comment:
Once this change is in a released version, it might be a good idea to also update https://github.com/goyek/x/blob/main/boot/boot.go.

Reply:
Will do after this PR is merged